src/containers/ProfileContainer/ProfileContainer.js

```diff
@@ -1,5 +1,11 @@
 import React, {Component} from 'react';
+import _ from 'lodash';
```

Comment:
Please add https://www.npmjs.com/package/babel-plugin-lodash to webpack config for such usage

Reply:
Done)